### PR TITLE
feat: add NFT picker

### PR DESCRIPTION
**Pull request by Sekhmet**
Closes: https://github.com/snapshot-labs/sx-ui/issues/153

Adds NFT picker to Send NFT modal.

**Note**: Sometimes Covalent is slow to fetch NFTs.

## Test plan

- Replace `loadNfts(spaces.pasta.wallets[0]);` with `loadNfts('0xfC00116B4049BBc8618847C74B44F0e103D571A3');` or other address you want to test with.
- You can pick, search and select NFTs.
- Preview shows up in the send NFT modal.

### CODE REVIEW

File: src/composables/useNfts.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        const title =
+          item.nft_data[0]?.external_data?.name ??
+          item.contract_name ??
+          'Untitled';
+        const displayTitle =
+          title.match(/(#[0-9]+)$/) || !id ? title : `${title} #${id}`;
+
+        const image =
+          item?.nft_data[0]?.external_data.image ??
+          `https://stamp.fyi/token/${item.contract_address}?s=256`;
```
Comment (Sekhmet) [inline src/composables/useNfts.ts:27]: Would be nice to have Stamp support for NFTs.

Comment (bonustrack) [reply src/composables/useNfts.ts:27]: Ok yes, feel free to add an issue for this, we could add support with url like this `stamp.fyi/nft/$address/$tokenId`. Also we should always use the cdn subdomain `cdn.stamp.fyi` to display image, at some point we will have a landing page at stamp.fyi. But currently I dont think we should use stamp here with the token url.

Comment (bonustrack) [reply src/composables/useNfts.ts:27]: Unless you are using it to get Blockie fallback image?

Comment (Sekhmet) [reply src/composables/useNfts.ts:27]: Here it's only to load the fallback (blockie) image in two cases:
1. image is missing on the NFT
2. image is there, but it fails to load ([here](https://github.com/snapshot-labs/sx-ui/blob/9809780a3f8a08381f8f10716e6bc169ee1abb28/src/components/NftPreview.vue#L7-L9)).

File: src/components/NftPreview.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<script setup>
+const props = defineProps({
+  item: Object,
+  size: Number
+});
+
+function handleError(e) {
+  e.target.src = `https://cdn.stamp.fyi/token/${props.item.contract_address}?s=256`;
```
Comment (bonustrack) [inline src/components/NftPreview.vue:8]: I'd recommend to use a 2x size here to get more precision and optimize to not load too big image
```suggestion
  e.target.src = `https://cdn.stamp.fyi/token/${props.item.contract_address}?s=${props.size * 2}`;
```

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**